graph-manager-docs/source/setup-analytics.md

```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
+The request can also optionally include a `Content-Type` header with value `application/protobuf`, but this is not required.
 
-Once a metrics report (i.e. batch of traces) is prepared, it will need to be sent to an ingress for aggregation and sampling. Currently, this is all performed in Apollo's cloud services. The endpoint for this aggregation and sampling is at `https://engine-report.apollodata.com/api/ingress/traces`, which supports the protobuf format mentioned above via a `POST` request. The reporting endpoint accepts a gzipped body as well. To see the full reference implementation, see the `sendReport()` method in the [TypeScript reference agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L210).
+For a reference implementation, see the `sendReport()` method in the [TypeScript reference agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L210).
 
-Reporting agents can authenticate either via the `X-Api-Key` header or the `authtoken` cookie with the service API key.
+#### Tuning reporting behavior
 
-We recommend implementing retries with backoff on 5xx responses and network errors and allowing for the batching size to be tunable by the user. Additionally, we recommend adding a shutdown hook to send all pending reports to ensure that healthy server shutdowns do not result in missing data, as this tracing information is especially important.
+We recommend implementing retries with backoff when you encounter `5xx` responses or networking errors when communicating with the reporting endpoint. Additionally, implement a shutdown hook to make sure you push all pending reports before your server initiates a healthy shutdown.
 
-> NOTE: In the future, we plan to release a local aggregation and sampling agent that could be used to lessen the bandwidth requirements on reporting agents.
+### Implementing additional reporting features
 
-### 4. [Optional] Advanced Reporting Features
+The reference TypeScript implementation includes several features that you might want to include in your implementation. All of these features are implemented in the agent itself, and are documented in the interface description for the `EngineReportingOptions` of [the agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L48).
 
-The reference TypeScript implementation also includes several more advanced features which may be worth porting to new implementations. All of these features are implemented in the agent itself and are documented in the interface description for the EngineReportingOptions of [the agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L51).
+For example, the option to send reports immediately is particularly useful for GraphQL servers that run in a serverless environment, like AWS Lambda or Google Cloud Functions.
```

Comment:
can we remove this line? We don't want to recommend that feature actually 😂 or replace it with
> For example, the option to modify variable values is useful in allowing reporting clients to obfuscate potentially private information while maintaining variable information that's useful in debugging.